### DISCUSSION TITLE
[SUGGESTION] Enter/Esc input difference for text fields

### DISCUSSION THREAD

**Discussion by lthoerner** [musescore/MuseScore#11794] [Ideas]
As a relatively amateur Musescore user, it has confused and peeved me slightly that, when entering a new value for a text field, the Enter key does not set it, but instead makes a line break. I would really love if the behavior would be Esc to cancel editing, Enter to confirm changes, and Shift-Enter to line break. This could be either a preference or a default behavior. Let me know what you think!

**Comment by HemantAntony** [comment]
Yeah I agree with this. I think GitHub and Discord does something similar

But the roles might be switched between the `Enter` and `Shift + Enter` shortcuts

**Comment by cbjeukendrup** [comment]
I'm afraid that many users will not easily discover something like Shift+Enter. 
In MS Word as well as Apple Pages, the behaviour in text boxes is similar to MuseScore's current behaviour. In those apps, it has slightly annoyed me too, but still I think it is the wiser choice, to keep it simple to understand for everyone.

**Reply by lthoerner** [reply]
This is sort of why I was thinking that it could be an option in settings for people who want it.
